Finmer.Game/Gameplay/GameSession.cs

```diff
@@ -61,6 +61,7 @@ public class GameSession : IDisposable
         private readonly Thread m_ScriptThread;
         private readonly AutoResetEvent m_ScriptWaitEvent;
         private bool m_ScriptThreadStop;
+        public bool gameover;
```

Comment:
- Please make sure to adhere to the [naming guidelines](https://github.com/pileofwolves/finmer/blob/master/.github/CONTRIBUTING.md#naming-conventions). The name should accurately describe what the field is for. As it is, I think it's a little misleading because this field does _not_ indicate that the game is actually over, only that a script has _requested_ for the game to end 'at the earliest convenience' (= at the end of the turn).
- This field should not be public. Right now, another class could set `gameover` not just to `true`, but also flip it back to `false`. What would that mean? That doesn't sound like something a random outsider should be able to do. Better to wrap it in a small public function that sets `gameover`, so that that is the only mutation an outsider can make use of.

```suggestion
        private bool m_GameOverRequested;
```